src/main/java/uk/gov/cabinetoffice/csl/controller/agencytoken/AgencyTokenVerificationController.java

```diff
@@ -144,6 +144,6 @@ private void buildGenericErrorModel(Model model, VerifyTokenForm form) {
     }
 
     private void addOrganisationsToModel(Model model) {
-        model.addAttribute("organisations", civilServantRegistryClient.getAllOrganisationsFromCache());
+        model.addAttribute("organisations", civilServantRegistryClient.getAllOrganisations());
```

Comment:
Could you please use CsrsService in this class instead of CivilServantRegistryClient? 
I think it would be better to use CsrsService in all the controllers instead of CivilServantRegistryClient.